pyctbgui/services/DACs.py

```diff
@@ -102,8 +102,8 @@ def setDAC(self, i):
         value = -100
         if checkBoxDac.isChecked():
             value = spinBox.value()
-
-        self.det.setDAC(dac, value, checkBoxmV.isChecked())
+        cond = checkBoxDac.isChecked() and checkBoxmV.isChecked()
```

Comment:
I know it's a nit-picky thing but could you change the name of cond (for example to in_mV) to better reflect what is happening when calling setDAC?